scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -356,7 +356,7 @@ private[scalatest] class RunnerJFrame(
                 </table>
                 <table>
                 <tr valign="top">
-                <td align="left" colspan="2">{ getHTMLForStackTrace(cause.getStackTrace.toList) }</td>
+                <td align="left" colspan="2">{ getHTMLForStackTrace(cause.getStackTrace.map(_.nn).toList) }</td>
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.